### PR TITLE
Some experimental changes

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

**Attribute and HP/SP rescaling is reverted. Temporarily or not, it may re-appear in future PRs, but not this one.**

A bunch of changes(organized by different commits):
- Overload success rate decreases are now personal; For WAW it's -3, for ALEPH -6.
- A lot of random "code improvements".

## Why It's Good For The Game

I decided it's time to do something about the game, something felt off, so it is time to figure what will improve it.

- This fixes the problem where if one agent spam-works on an abnormality it punishes everyone, but them. With personal overload chances it punishes ONLY the person who decided to work 5 times on an abno.
- Neat code.

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
Issuing a DNM until our devs sit down and have a talk about it.

**Comment by Kitsunemitsu** [comment]
This PR CANNOT Be merged before 
Sunday, 17 September 2023 21:00 GMT due to the release of W-Corp.

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -1238,8 +1238,8 @@
 
 /mob/living/carbon/human/updatehealth()
 	if(LAZYLEN(attributes))
-		maxHealth = 100 + round(get_modified_attribute_level(src, FORTITUDE_ATTRIBUTE))
-		maxSanity = 100 + round(get_modified_attribute_level(src, PRUDENCE_ATTRIBUTE))
+		maxHealth = DEFAULT_HUMAN_MAX_HEALTH + round(get_attribute_level(src, FORTITUDE_ATTRIBUTE) * FORTITUDE_MOD + get_level_bonus(src, FORTITUDE_ATTRIBUTE))
+		maxSanity = DEFAULT_HUMAN_MAX_SANITY + round(get_attribute_level(src, PRUDENCE_ATTRIBUTE) * PRUDENCE_MOD + get_level_bonus(src, PRUDENCE_ATTRIBUTE))
```
Comment (LanceSmites328) [inline code/modules/mob/living/carbon/human/human.dm:1242]: Are fortitude and prudence bonuses to HP and SP not affected by the modifier?

**Review by LanceSmites328 — DISMISSED** [review]

File: code/modules/mob/living/carbon/human/human.dm (hunk 1 above)
Comment (Coxswain-Navigator) [reply code/modules/mob/living/carbon/human/human.dm:1242]: Level_bonus should have the same weight as ordinary attributes.

Comment (EgorDinamit) [reply code/modules/mob/living/carbon/human/human.dm:1242]: The bonuses are separate number, as shown in the "show attributes" menu.

**Review by Kitsunemitsu — APPROVED** [review]
This PR is approved.

**Pull request merged**